src/msg/Messenger.h

```diff
@@ -472,6 +472,7 @@ class Messenger {
    * @return 0 on success, or -errno on failure.
    */
   virtual int send_message(Message *m, const entity_inst_t& dest) = 0;
+  virtual int send_message(Message *m, Connection *con) {return 0;}
```

Comment:
this is from an old version of the Messenger interface, i don't think we want to add it back

if the commented-out line in Dispatcher::send_request() isn't working (was it sending to the wrong dispatcher?), we'll need to restructure things in DirectMessenger/DirectConnection so it does
```c++
//  conn.get()->send_message(m);
```